importexport/musicxml/exportxml.cpp

```diff
@@ -2403,21 +2419,38 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
       // and finally the attributes whose elements are children of <technical>
       for (const Articulation* a : na) {
             auto sid = a->symId();
+            QString placement = "";
+            QString direction = "";
 
-            if (sid == SymId::stringsHarmonic) {
+            QString attr = QString("");
+            if (!a->isStyled(Pid::ARTICULATION_ANCHOR) && a->anchor() != ArticulationAnchor::CHORD) {
+                  placement = (a->anchor() == ArticulationAnchor::BOTTOM_STAFF || a->anchor() == ArticulationAnchor::BOTTOM_CHORD) ? "below" : "above";
+                  }
+            else if (!a->isStyled(Pid::DIRECTION) && a->direction() != Direction::AUTO) {
+                  direction = (a->direction() == Direction::DOWN) ? "down" : "up";
+                  }
+            /* For future use if/when implemented font details for articulation
+            if (!a->isStyled(Pid::FONT_FACE))
+                  attr += QString(" font-family=\"%1\"").arg(a->getProperty(Pid::FONT_FACE).toString());
+            if (!a->isStyled(Pid::FONT_SIZE))
+                  attr += QString(" font-size=\"%1\"").arg(a->getProperty(Pid::FONT_SIZE).toReal());
+            if (!a->isStyled(Pid::FONT_STYLE))
+                  attr += fontStyleToXML(static_cast<FontStyle>(a->getProperty(Pid::FONT_STYLE).toInt()), false);
+            */
+
+            auto mxmlTechn = symIdToTechn(sid);
+            if (mxmlTechn != "") {
                   notations.tag(_xml);
                   technical.tag(_xml);
-                  _xml.stag("harmonic");
-                  _xml.tagE("natural");
-                  _xml.etag();
-                  }
-            else {
-                  auto mxmlTechn = symIdToTechn(sid);
-                  if (mxmlTechn != "") {
-                        notations.tag(_xml);
-                        technical.tag(_xml);
-                        _xml.tagE(mxmlTechn);
+                  if (sid == SymId::stringsHarmonic) {
+                        if (placement != "")
+                              attr += QString(" placement=\"%1\"").arg(placement);
+                        _xml.stag(mxmlTechn + attr);
+                        _xml.tagE("natural");
+                        _xml.etag();
                         }
+                  else // TODO: check additional modifier (attr) for other symbols
```

Comment:
Is there any reason not to do this right away? 
I mean, TODO can hang VERY long, we have many such TODOs throughout the project

Reply:
basically not to increase complexity, and because it goes even further outside the scope of the PR